src/historyevent.ts

```diff
@@ -2,14 +2,19 @@ import { HistoryEventType } from "./classes";
 
 export class HistoryEvent {
```

Comment:
I see a pattern of making wrapper classes with an any object inside of it. Might be better to make these wrappers Generics so HistoryEvent<T> with input having type T.